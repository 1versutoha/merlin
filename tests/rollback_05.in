["reset","name","test.ml"]
["tell","struct","let x = \"hello\"\nmodule A = struct\n  let x = 5\nend\nlet _ = x\n"]
["tell","end",null]
["type","enclosing",{"expr":"x","offset":0},{"line":5,"col":8}]
["seek","before",{"line":5,"col":0}]
["tell","struct","\nlet _ = x\n"]
["tell","end",null]
["type","enclosing",{"expr":"x","offset":0},{"line":5,"col":8}]
