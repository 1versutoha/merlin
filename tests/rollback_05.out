["return",{"line":1,"col":0}]
["return",false]
["return",true]
["return",[{"start":{"line":5,"col":8},"end":{"line":5,"col":9},"type":"string"},{"start":{"line":5,"col":8},"end":{"line":5,"col":9},"type":"string"}]]
["return",{"line":4,"col":3}]
["return",false]
["return",true]
["return",[{"start":{"line":5,"col":8},"end":{"line":5,"col":9},"type":"string"},{"start":{"line":5,"col":8},"end":{"line":5,"col":9},"type":"string"}]]
